### PR TITLE
[CDAP-17357] Replication add ability to drop table from assessment

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-17357
Build: https://builds.cask.co/browse/CDAP-UDUT852

Requires backend change: https://github.com/data-integrations/delta/pull/74

### CODE REVIEW

File: cdap-ui/server/config/themes/default.json [hunk 1]
```diff
@@ -50,6 +50,6 @@
     "namespace-security": true,
     "create-profile": true,
     "reload-system-artifacts": true,
-    "cdc": false
+    "cdc": true
```
Comment (ajainarayanan) [inline cdap-ui/server/config/themes/default.json:53]: nit: Is this intentional? Do we want to surface CDC by default in OSS UI?

**Review by ajainarayanan — APPROVED** [review]
Minor commment. 👍 LGTM

File: cdap-ui/server/config/themes/default.json (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/server/config/themes/default.json:53]: Yes this is intentional

**Pull request merged**